src/presentation/components/ReCaptcha/useReCaptcha.ts

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from 'react';
```

Comment:
@gastonri  What do you think about moving this file to src/app/hooks? as you don't have a component for reCaptcha you don't need this here inside the components folder

Reply:
Yeah. It makes total sense.